### DISCUSSION TITLE
Right clicking icons in bottom taskbar should allow to toggle 'Always on Top' mode

### DISCUSSION THREAD

**Discussion by RustoMCSpit** [linuxmint/discussions#243] [Ideas]
Software can be pinned to the top by clicking on the app, it should also be possible by its icon

**Comment by superian** [comment]
YES!! This is the one thing I miss from MATE: the ability to set a window to be 'always on top' from the taskbar. 

In Cinnamon, you can move a window's workspace or set it to be visible on all workspaces from the taskbar, as in MATE, but to have it 'always on top' needs you to right click on the window's title bar and set it there.

Two use cases here: the one I'm most used to is that if you have a large / maximised 'always on top' window, you need to minimise / resize that one before you can see the second window's title bar to set that one. (I quite often need one small window on top of a section of another 'on top' one.)

Today, I discovered that under Cinnamon, the current version of Zoom doesn't have a title bar for its meeting window, so I can't have it 'on top' at all. Other such programs exist.

**Reply by klangman** [reply]
> YES!! This is the one thing I miss from MATE: the ability to set a window to be 'always on top' from the taskbar.

Just saw this now, seems like a reasonable request so I added this feature to my window list applet (https://github.com/klangman/CassiaWindowList).

1. Added a window-list button context menu item to toggle the "Always on top" state.
2. Added mouse action options to toggle "Always on top" state so you can configure a mouse button to perform this action.

You would have to switch over to using my window-list applet, hopefully it meets your needs, but if not you can always switch back to the default window list.

Currently this addition is only in my github version, but I will push to the changes to the cinnamon spices version at some point when I have some time. When I do, it will be available from the Cinnamon System Setting Applets tool and from here: https://cinnamon-spices.linuxmint.com/applets/view/372, until then you would need manually install my github version if you are interested in trying it out right now.

**Comment by klangman** [comment]
CassiaWindowList 2.3.1 on Cinnamon Spices now has the "Always on top" features as well.
